deployment/centos-package-x64/Dockerfile

```diff
@@ -13,13 +13,12 @@ RUN yum update -y \
  && yum install -y epel-release
 
 # Install build dependencies
-RUN yum install -y @buildsys-build rpmdevtools yum-plugins-core libcurl-devel fontconfig-devel freetype-devel openssl-devel glibc-devel libicu-devel wget git
+RUN yum install -y @buildsys-build rpmdevtools yum-plugins-core libcurl-devel fontconfig-devel freetype-devel openssl-devel glibc-devel libicu-devel git
 
 # Install recent NodeJS and Yarn
-RUN wget -O- https://raw.githubusercontent.com/creationix/nvm/v0.35.0/install.sh | /bin/bash \
- && source "$HOME/.nvm/nvm.sh" \
- && nvm install v8 \
- && npm install -g yarn
+RUN curl -fSsLo /etc/yum.repos.d/yarn.repo https://dl.yarnpkg.com/rpm/yarn.repo \
+ && rpm -i https://rpm.nodesource.com/pub_8.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm \
+ && yum install -y yarn
```

Comment:
Why this uses yarn when our other packages moved to npm?

Reply:
It probably should follow the others in using NPM to install Yarn, but for quickness in getting this working I don't want to mess with it any further.